docs/source/api/server_server_capabilities.rst

```diff
@@ -0,0 +1,223 @@
+:serverId:         The server's id
+:lastUpdated:      The date and time at which this association between the server and the :term:`Server Capability` was last updated, in an ISO-like format
+:serverCapability: The :term:`Server Capability`'s name
+
+.. code-block:: http
+	:caption: Response Example
+
+	HTTP/1.1 200 OK
+	Access-Control-Allow-Credentials: true
+	Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept, Set-Cookie, Cookie
+	Access-Control-Allow-Methods: POST,GET,OPTIONS,PUT,DELETE
+	Access-Control-Allow-Origin: *
+	Content-Type: application/json
+	Set-Cookie: mojolicious=...; Path=/; HttpOnly
+	Whole-Content-Sha512: eQrl48zWids0kDpfCYmmtYMpegjnFxfOVvlBYxxLSfp7P7p6oWX4uiC+/Cfh2X9i3G+MQ36eH95gukJqOBOGbQ==
+	X-Server-Name: traffic_ops_golang/
+	Date: Mon, 07 Oct 2019 22:15:11 GMT
+	Content-Length: 157
+
+	{
+		"alerts": [
+			{
+				"text": "server server_capability was created.",
+				"level": "success"
+			}
+		],
+		"response": {
+			"lastUpdated": "2019-10-07 22:15:11+00",
+			"serverId": 1,
+			"serverCapability": "disk"
+		}
+	}
+
+``DELETE``
+==========
+Disassociate a server from a :term:`Server Capability`
+
+:Auth. Required: Yes
+:Roles Required: "admin" or "operations"
+:Response Type:  Object
+
+Request Structure
+-----------------
+.. table:: Request Query Parameters
+
+	+------------------+----------+------------------------------------------------------------------+
+	| Name             | Required | Description                                                      |
+	+==================+==========+==================================================================+
+	| serverId         | yes      | ID of the server to disassociate                                 |
+	+------------------+----------+------------------------------------------------------------------+
+	| serverCapability | yes      | :term:`Server Capability` name to disassociate from given server |
+	+------------------+----------+------------------------------------------------------------------+
+
+.. code-block:: http
+	:caption: Request Example
+
+	DELETE /api/1.4/server_server_capabilities?serverId=1&serverCapability=disk HTTP/1.1
+	Host: trafficops.infra.ciab.test
+	User-Agent: curl/7.47.0
+	Accept: */*
+	Cookie: mojolicious=...
+
+Response Structure
+------------------
+
+.. code-block:: http
+	:caption: Response Example
+
+	HTTP/1.1 200 OK
+	Access-Control-Allow-Credentials: true
+	Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept, Set-Cookie, Cookie
+	Access-Control-Allow-Methods: POST,GET,OPTIONS,PUT,DELETE
+	Access-Control-Allow-Origin: *
+	Content-Type: application/json
+	Set-Cookie: mojolicious=...; Path=/; HttpOnly
+	Whole-Content-Sha512: UFO3/jcBFmFZM7CsrsIwTfPc5v8gUiXqJm6BNp1boPb4EQBnWNXZh/DbBwhMAOJoeqDImoDlrLnrVjQGO4AooA==
+	X-Server-Name: traffic_ops_golang/
+	Mon, 07 Oct 2019 22:15:11 GMT
+	Content-Length: 96
+
+	{
+		"alerts": [
+			{
+				"text": "server server_capability was deleted.",
```

Comment:
Is this just an example or will the alert be more like "CACHE_MEMORY server capability removed from hostname"

Reply:
some comment for POST

Reply:
this is the exact text that will be shown which I am guessing is consistent across a lot of endpoints as I am using the shared handlers to CREATE/DELETE/READ and them it just puts the type in the response message https://github.com/apache/trafficcontrol/blob/master/traffic_ops/traffic_ops_golang/api/shared_handlers.go#L300 https://github.com/apache/trafficcontrol/blob/master/traffic_ops/traffic_ops_golang/api/shared_handlers.go#L356. I can look at other endpoints to see if they handled the GetType differently and put in specific object details or left that up to https://github.com/apache/trafficcontrol/blob/master/traffic_ops/traffic_ops_golang/api/shared_interfaces.go#L47